packages/rancher-istio/package.yaml

```diff
@@ -1,3 +1,2 @@
 url: local
-packageVersion: 0
-doNotRelease: true
\ No newline at end of file
+version: 100.0.3+up1.10.4
```

Comment:
1. This chart is removed from dev-2.6 in this [PR](https://github.com/rancher/charts/pull/1600#pullrequestreview-800660559), so it is not a backporting anymore, right? 
2. Do we decide to use the `100.x.x` version schema for Istio in the v2.5 line?

Reply:
1. You are correct, because the version of 1.10.4 in the dev-v2.6 branch has the kiali server api bug in it, we did not want to back port it as is. We still wanted to have 1.10.4 in the dev-v2.5 branch, but with an istio-installer version that did not have the kiali server api issue.
2. We did not decide - I had failed to change to the version back to the v2.5 line versioning, will update PR and re-request review.